### PR TITLE
README.md for logo directory

### PR DESCRIPTION
**Pull request by bockthom**
### Prerequisites

- [x] I adhere to the coding conventions (described [here](https://github.com/se-passau/coronet/blob/master/CONTRIBUTING.md)) in my code.
- [x] I have updated the copyright headers of the files I have modified.
- [x] I have written appropriate commit messages, i.e., I have recorded the goal, the need, the needed changes, and the location of my code modifications for each commit. This includes also, e.g., referencing to relevant issues.
- [x] I have put signed-off tags in *all* commits.
- [x] I have updated the changelog file [NEWS.md](https://github.com/se-passau/coronet/blob/master/NEWS.md) appropriately.
- [x] The pull request is opened against the branch `dev`.

### Description

Today, we noticed that the 'svg' files of our logo are wrongly displayed if the used font *Larabie Font* is not installed. This might cause troubles if someone does not notice that the font is wrongly displayed and also may not know what the reason for that is. Therefore, we should provide this information in our repository.

As the 'svg' files of our logo are wrongly displayed if the used font
'Larabie Font' is not installed on a specific system, add some further
information which font needs to be installed to correctly display the
font in the 'svg' file.

@ChristianKaltenecker proposed the idea to add a separate `README.md` file in the logo directory, containing additional information on logo and font, and how to install the font. As I like his idea, I've created such a `logo/README.md` and added the respective information. (Props to @ChristianKaltenecker for providing the links!)

@clhunsen, @hechtlC, @ChristianKaltenecker: Please have a look at the new `logo/README.md` and let me know whether you think that the information there is helpful and correct, or whether you would like to add more information there.

@clhunsen @hechtlC: I have added the new file also to our website config, so that it can also be displayed on GitHub pages. However, at the moment there is no link to this page from the main website yet. Would you like to add a link from the main website to the logo website (i.e., add a new section directly before the last section), or do you think it is not necessary to link the new page and it would be rather confusing to add a link from the main website to the logo README? I would be glad if you would provide me with your thoughts on that.

(While adding the new `logo/README.md`, I noticed that URLs are correctly converted to links on GitHub Markdown, but not on our to-be-published-soon auto-generated website. Therefore, I converted all URLs into links, so that they will be correctly displayed on our website.)

This (hopefully) is the last question related issue #167. Looking forward to the new release soon, only 3 more pull requests...

### CODE REVIEW

**Review by hechtlC — APPROVED** [review]
Thanks for the additions @bockthom .
I don't see any issues with this so far and as soon as the changes we talked about earlier are incorporated, this can be merged.

**Review by ChristianKaltenecker — APPROVED** [review]
Looks good to me!
@bockthom Thanks for incorporating that feedback.

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ All `R` tools and IDEs should provide a  more sophisticated interface for the in
 #### Folder structure of the input data
 
 To use this network library, the input data has to match a certain folder structure and agree on certain file names.
-The data folder – which can result from consecutive runs of `Codeface` [https://github.com/se-passau/codeface] (branch `infosaar-updates`) and `codeface-extraction` [https://github.com/se-passau/codeface-extraction] – needs to have the following structure (roughly):
+The data folder – which can result from consecutive runs of `Codeface` [[https://github.com/se-passau/codeface](https://github.com/se-passau/codeface)] (branch `infosaar-updates`) and `codeface-extraction` [[https://github.com/se-passau/codeface-extraction](https://github.com/se-passau/codeface-extraction)] – needs to have the following structure (roughly):
```
Comment (clhunsen) [inline README.md:74]: Both repositories are already migrated to `se-sic`. Do you want to adjust the links to reflect the migration? (Same applies to `codeface-extraction` in Line 9.)

**Review by clhunsen — COMMENTED** [review]
Nice idea with the README file for the logo variants. I approve the changes with great pleasure but have one tiny question: Do you want to adjust all links to `se-sic` where possible (see my comment)?

File: README.md (hunk 1 above)
Comment (bockthom) [reply README.md]: Done (right now) :wink:

**Review by clhunsen — APPROVED** [review]

**Comment by bockthom** [comment]
Thanks @clhunsen for your comments. 

I just spotted a minor inconsistency in the description of our folder structure in our `README.md`. So, I've added an additional commit right now. But now the PR should be finished and ready for being merged :wink:

**Review by clhunsen — APPROVED** [review]

**Pull request merged**